### PR TITLE
Raspibolt/bonus tor

### PR DESCRIPTION
**Pull request by BobleChinois**
This PR is a bonus to the raspibolt guide that explain why you might want to run Bitcoin and LND through Tor, and how to do it.

### CODE REVIEW

**Comment by Stadicus** [comment]
Thanks for this great bonus guide. I'm in the process of reviewing it and also doing some wording / layout changes. I also adapted some steps to have the same handling as other parts (eg. using systemd start / stop) and using full length paths for all config files.

One question: why do you use the `bionic` tor repository, as the RaspiBolt guide is based on the Raspbian Stretch?

**Comment by BobleChinois** [comment]
Because my guide doesn't use a Pi, and I choose Lubuntu, hence the `bionic` repository for Tor. Sorry I forgot to change that.

File: raspibolt/raspibolt_69_tor.md [hunk 1]
```diff
@@ -0,0 +1,250 @@
...
+  ```
+
+* Archive the current logfile and restart Bitcoin Core to use the adjusted configuration.
+  ```
+  $ sudo mv /home/bitcoin/.bitcoin/debug.log /home/bitcoin/.bitcoin/debug.log.old
+  $ sudo systemctl start bitcoind
+  ```
+
+#### Validation  
+Bitcoin Core is starting and we now need to if all connections are truly routet over Tor.
```
Comment (BobleChinois) [inline raspibolt/raspibolt_69_tor.md]: we now need to _check_ if all connections are truly _routed_ over Tor

File: raspibolt/raspibolt_69_tor.md [hunk 2]
```diff
@@ -0,0 +1,250 @@
...
+  
+  If the result is negative, just try again a minute later as sometimes the lookup will fail to contact your node for some reasons. If your node is persistently not reachable, verify your [port forwarding](raspibolt_20_pi.md#port-forwarding--upnp) and [firewall](raspibolt_20_pi.md#enabling-the-uncomplicated-firewall) settings.
+  
+* Check the IP address that other peers use to connect to your node. First, get your real public IP address and then verify that it is not used as `localaddr` by Bitcoin.
+  ```
+  $ curl icanhazip.com
+  $ bitcoin-cli getpeerinfo | grep  local
+  ```
+  
+  :warning: If you still see your real public IP address listed, something is wrong as one of your peers are currently connected with you on clearnet. This means that you're effectively deanonymized.
```
Comment (BobleChinois) [inline raspibolt/raspibolt_69_tor.md]: as one _or many_ of your peers are currently connected

**Comment by BobleChinois** [comment]
You don't need to apologize, it's way better now, thanks! 

I identified a couple typos, but once they are corrected we should be good to go.

You helped me so much to set up my own node, I'm very glad I can give a little back for all you have done!

**Comment by starkana** [comment]
It also says to use Tor3.6.6+ but such version doesn't even exist. The latest version is **[0.3.5.5-alpha - 2018-11-16](https://gitweb.torproject.org/tor.git/plain/ChangeLog)** 


I followed the Tor LND instructions and it says **lncli getinfo** or **lncli getnodeinfo [YOUR_PUBKEY]** should not display my public IP.  But I see 2 URI fields, one with an onion address the other with my public IPv4.  Is this correct?

It also says to use Tor3.6.6+ but such version doesn't even exist. The latest version is **[0.3.5.5-alpha - 2018-11-16](https://gitweb.torproject.org/tor.git/plain/ChangeLog)** Is this correct? I run **Tor 0.3.4.9 (git-074ca2e0054fded1)** and it seems to be running with Tor.

**Comment by BobleChinois** [comment]
Sorry I must have made a mistake with Tor versions, just checked [this page](https://github.com/lightningnetwork/lnd/blob/master/docs/configuring_tor.md) and in fact it is 0.3.3.6. My bad.

As for the 2 URI fields, I'm a bit puzzled I didn't even knew you could have 2 URIs...
Definitely doesn't feel right to me. Did your node had open channels on clearnet, and you upgraded to tor without closing existing channels? Just asking.

**Pull request merged**